gradle.properties

```
@@ -11,7 +11,7 @@ POM_LICENCE_DIST=repo
 POM_DEVELOPER_ID=mapbox
 POM_DEVELOPER_NAME=Mapbox
 
-POM_SNAPSHOT_VERSION_NAME=2.4.0-SNAPSHOT
+SNAPSHOT_VERSION_NAME=2.4.0-SNAPSHOT
```

```suggestion
VERSION_NAME=2.4.0-SNAPSHOT
```
To resolve the issue with `sed`.